integration/rest.go

```diff
@@ -0,0 +1,53 @@
+package integration
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+type blockResp struct {
+	BlockID struct {
+		Hash string `json:"hash"`
+	} `json:"block_id"`
+	Block struct {
+		Header struct {
+			Height string `json:"height"`
+		} `json:"header"`
+	} `json:"block"`
+}
+
+func QueryLatestBlock(endpoint string) ([]byte, int64, error) {
+	resp, err := http.Get(fmt.Sprintf("%s/blocks/latest", endpoint))
+	if err != nil {
+		return nil, 0, fmt.Errorf("failed to execute HTTP request: %w", err)
+	}
+	defer resp.Body.Close()
```

Comment:
I'm thinking of changing this logic to use gRPC instead of REST, but let's go with this code for now. I'm gonna find a nicer way when designing the panacea-e2e.